test/exports.test.ts

```diff
@@ -10,7 +10,9 @@ describe("Exports", function () {
     };
 
     options.enabled = false;
+    // @ts-expect-error
     options.onRateLimit(10, {}, {} as Octokit, 0);
+    // @ts-expect-error
     options.onSecondaryRateLimit(10, {}, {} as Octokit, 0);
   });
```

Comment:
Should we pass more significant `request` objects as a second argument instead of adding `// @ts-expect-error` as part of this PR?

Reply:
I don't think we need to here. This only tests that the method is available and exported.

The other tests handle it's proper usage